src/drawers/rectDrawer.ts

```diff
@@ -13,7 +13,7 @@ export module _Drawer {
 
     constructor(key: string, isVertical: boolean) {
       super(key);
-      this.svgElement("rect");
```

Comment:
Honestly, I'm not sure why this method exists anyway. Is there even any point to being able to change the SVG element on the fly?